src/sinks/mod.rs

```diff
@@ -123,6 +123,7 @@ pub enum HealthcheckError {
 #[configurable_component]
 #[allow(clippy::large_enum_variant)]
 #[derive(Clone, Debug)]
+#[allow(clippy::large_enum_variant)]
```

Comment:
Duplicate of the same thing two lines up.. :P

Reply:
Hah, merge glitch, will fix.